src/main/java/com/cozyhome/onlineshop/userservice/controller/UserSecuredController.java

```diff
@@ -65,11 +65,12 @@ public ResponseEntity<UserInformationResponse> getUserInfo(HttpServletRequest re
     @Operation(summary = "Logout.", description = "Logout.")
     @ApiResponses(value = {
             @ApiResponse(responseCode = SwaggerResponse.Code.CODE_200, description = SwaggerResponse.Message.CODE_200_FOUND_DESCRIPTION) })
+    @Secured({"ROLE_CUSTOMER", "ROLE_MANAGER", "ROLE_ADMIN"})
     @GetMapping("/logout")
     public ResponseEntity<String> logout(HttpServletRequest request) {
         String jwtToken = jwtTokenUtil.resolveToken(request);
         tokenBlackListService.saveTokenToBlackList(jwtToken);
-        log.warn("[ON UserSecuredController:logout]:: JwtToken {} was added to Black List.", jwtToken);
+        log.warn("[ON logout] :: JwtToken {} was added to Black List.", jwtToken);
```

Comment:
I suggest that in the logs you specify the class in which the method is located, this will help us to find this method faster

Reply:
No need to specify a class name in a log message, it already shown in logs.